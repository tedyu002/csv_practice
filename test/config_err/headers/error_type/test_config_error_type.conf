CSV_INPUT=test.input
CSV_OUTPUT=test.out
CSV_ERROR=test.err
CSV_RESULT=test.res
HEADERS=H1 INTEGE, H2 BOOL
