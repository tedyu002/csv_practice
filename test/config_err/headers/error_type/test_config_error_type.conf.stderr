Failed to parse HEADERS 'H1 INTEGE, H2 BOOL'.
config 'test_config_error_type.conf' parse error
